File: akka-serialization-jackson/src/test/java/jdoc/akka/serialization/jackson/v1/OrderAdded.java
/*
 * Copyright (C) 2019-2024 Lightbend Inc. <https://www.lightbend.com>
 */

package jdoc.akka.serialization.jackson.v1;

import akka.serialization.jackson.JsonSerializable;

// #rename-class
public class OrderAdded implements JsonSerializable {
  public final String shoppingCartId;

  public OrderAdded(String shoppingCartId) {
    this.shoppingCartId = shoppingCartId;
  }
}
// #rename-class

File: akka-serialization-jackson/src/test/scala/doc/akka/serialization/jackson/v1/OrderAdded.scala
/*
 * Copyright (C) 2019-2024 Lightbend Inc. <https://www.lightbend.com>
 */

package doc.akka.serialization.jackson.v1

import akka.serialization.jackson.JsonSerializable

// #rename-class
case class OrderAdded(shoppingCartId: String) extends JsonSerializable
// #rename-class
